components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -878,13 +878,19 @@ - (void)fhv_contentOffsetDidChange {
 #if DEBUG
 - (void)fhv_scrollViewDidPan:(UIPanGestureRecognizer *)pan {
   if (pan.state == UIGestureRecognizerStateEnded && [self fhv_canShiftOffscreen]) {
+    // TODO(#1254): Re-enable sanity check assert on viewDidPan
+    // Remove #if 0 and else clause, keep NSAssert() to re-enable
+#if 0
     // You _must_ implement the target content offset method in your UIScrollViewDelegate.
     // Not implementing the target content offset method can allow the status bar to get into an
     // indeterminate state and may cause your app to be rejected.
     NSAssert(_didAdjustTargetContentOffset, @"%@ isn't invoking %@'s %@.",
              NSStringFromClass([_trackingScrollView class]), NSStringFromClass([self class]),
              NSStringFromSelector(
                  @selector(trackingScrollViewWillEndDraggingWithVelocity:targetContentOffset:)));
+#else
+    ;
```

Comment:
I was concerned that an if statement without a executed block might get optimized away.  If the if statement goes away, do we lose any expected side effects of calling [self fhv_canShiftOffscreen]?  As _didAdjustTargetContentOffset get set as a side effect of some other methods, this was a concern.

I've tried to simplify.  Let me know if still isn't clear.